pages/artykuly/_categoryCode/index.vue

```diff
@@ -35,19 +35,22 @@ export default {
   },
   computed: {
     ...mapGetters({
-      categoryCodeList: 'view/articles/categoryCodeList',
+      categoryArticlesCodeList: 'view/articles/categoryArticlesCodeList',
       isMobile: 'general/isMobile',
     }),
-    list () {
-      return _.map(this.categoryCodeList, code => this.$store.getters['resources/articles'][code])
+    articlesList () {
+      return _.map(this.categoryArticlesCodeList, code => this.$store.getters['resources/articles'][code])
     },
   },
   fetch ({ store, params }) {
-    if (params.categoryCode === 'wszystkie') {
-      return store.dispatch('view/articles/getMainList')
-    }
+    const promises = [
+      store.dispatch('view/categories/getCategories'),
+      params.categoryCode === 'wszystkie' ?
```

Comment:
'wszystkie' as const CATEGORY_CODE_ALL ?

Reply:
This will be changed in PR which Kamil Stasiak prepares. We managed to not use if in that area at all ;)

Reply:
it*